src/zsock.c

```diff
@@ -1912,17 +1921,17 @@ zsock_test (bool verbose)
     assert (port >= 60000 && port <= 60500);
 
     //  Test zsock_attach method
-    zsock_t *server = zsock_new (ZMQ_DEALER);
```

Comment:
Sorry to be a pain in the back but CI compilation is still broken:

```
src/zsock.c: In function 'zsock_test':
src/zsock.c:2111:5: error: 'server' undeclared (first use in this function)
     server = zsock_new (ZMQ_SERVER);
```

There's more usage of that variable below, so perhaps it's easier to use different names in the new snippet? Up to you though

Reply:
No problem. Fixed and running `make && make check` this time. Stay tuned...

Reply:
Okay, this time it should be good. Sorry about this, I'm too used to Ruby. 😆